ocp_resources/virtual_machine_instance.py

```diff
@@ -124,6 +138,9 @@ def wait_until_running(self, timeout=TIMEOUT_4MINUTES, logs=True, stop_status=No
 
             virt_pod = self.virt_launcher_pod
             if virt_pod:
+                self.logger.error(
+                    f"Status of virt-launcher pod {virt_pod.name}: {virt_pod.status}"
```

Comment:
Please also log here vm and vmi names
can be something like:
`Status of virt-launcher pod {virt_pod.name}: {virt_pod.status} [VM: ......, VMI: .....]`

Reply:
added vmi name + status in exception block